src/multiversioning.zig

```diff
@@ -233,12 +279,21 @@ pub const MultiversionHeader = extern struct {
             }
 
             for (sizes) |s| if (s == 0) return error.InvalidPastReleases;
+            for (flags) |f| if (f.padding != 0) return error.InvalidPastReleases;
+            for (git_commits) |g| if (std.mem.allEqual(u8, &g, 0)) return error.InvalidPastReleases;
+
+            for (release_client_mins) |v| if (v == 0) return error.InvalidPastReleases;
+            if (!std.sort.isSorted(u32, release_client_mins, {}, std.sort.asc(u32))) {
+                return error.InvalidPastReleases;
+            }
         }
     };
 
     /// Covers MultiversionHeader[@sizeOf(u128)..].
     checksum_header: u128 = undefined,
 
+    schema_version: u32 = 1,
```

Comment:
https://github.com/tigerbeetle/tigerbeetle/pull/2013/commits/f2a921c147b7e5ecc41cdfaf45aad418b5a9f89a#r1679274816 (...) But with an explicit schema version, perhaps it's safer to not allow smuggling of things, and get rid of the `reserved` here entirely. We can do what's mentioned in https://github.com/tigerbeetle/tigerbeetle/pull/2013#discussion_r1679112479 for any changes.